test/integration-tests/IntegrationTests.MongoDB/MongoDbCollection.cs

```diff
@@ -68,7 +69,7 @@ private TestcontainersContainer LaunchMongoContainer(int port)
                 : Wait.ForUnixContainer();
 
             var mongoContainersBuilder = new TestcontainersBuilder<TestcontainersContainer>()
-              .WithImage("mongo")
+              .WithImage(MongoDbImage)
```

Comment:
nit: leaving a comment here will help understand why we use the hard-coded version.

Reply:
I think that actually leaving it without a fix version was a bug, that caused uncontrollable environment.
Better way should be integrating it with dependabot somehow, so when PR is opened with version update, we can actually see what it causes.